server/src/server/keyspace/KeyspaceManager.java

```diff
@@ -18,149 +18,29 @@
-        try (TransactionOLTP tx = systemKeyspaceSession.transaction().write()) {
-            AttributeType<String> keyspaceName = tx.getSchemaConcept(KEYSPACE_RESOURCE);
-            Attribute<String> attribute = keyspaceName.attribute(keyspace.name());
-            if (attribute == null) {
-                throw GraknServerException.create("It is not possible to delete keyspace [" + keyspace.name() + "] as it does not exist.");
-            }
-            Thing thing = attribute.owner();
-            thing.delete();
-            attribute.delete();
-
-            existingKeyspaces.remove(keyspace);
-            tx.commit();
-        }
+    private final Cluster storage;
+    private final Set<String> internals = new HashSet<>(Arrays.asList("system_traces", "system", "system_distributed", "system_schema", "system_auth"));
+    
+    public KeyspaceManager(int storageNativeTransportPort) {
+        storage = Cluster.builder().addContactPoint("localhost").withPort(storageNativeTransportPort).build(); // TODO: don't hardcode ip and port
     }
 
     public Set<KeyspaceImpl> keyspaces() {
-        try (TransactionOLTP graph = systemKeyspaceSession.transaction().write()) {
-            AttributeType<String> keyspaceName = graph.getSchemaConcept(KEYSPACE_RESOURCE);
-
-            return graph.<EntityType>getSchemaConcept(KEYSPACE_ENTITY).instances()
-                    .flatMap(keyspace -> keyspace.attributes(keyspaceName))
-                    .map(name -> (String) name.value())
-                    .map(KeyspaceImpl::of)
-                    .collect(Collectors.toSet());
-        }
-    }
-
-    public void loadSystemSchema() {
-        Stopwatch timer = Stopwatch.createStarted();
-        try (TransactionOLTP tx = systemKeyspaceSession.transaction().write()) {
-            if (tx.getSchemaConcept(KEYSPACE_ENTITY) != null) {
-                LOG.info("System schema has been previously loaded");
-                return;
-            }
-            LOG.info("Loading schema");
-            loadSystemSchema(tx);
-            tx.commit();
-            LOG.info("Loaded system schema to system keyspace. Took: {}", timer.stop());
-        } catch (RuntimeException e) {
-            LOG.error("Error while loading system schema in {}. The error was: {}", timer.stop(), e.getMessage(), e);
-            throw e;
-        }
-    }
-
-    /**
-     * Loads the system schema using the provided Transaction.
-     */
-    private void loadSystemSchema(TransactionOLTP tx) {
-        AttributeType<String> keyspaceName = tx.putAttributeType("keyspace-name", AttributeType.DataType.STRING);
-        tx.putEntityType("keyspace").key(keyspaceName);
+        Set<String> result = storage.connect().getCluster().getMetadata().getKeyspaces().stream()
+                .map(KeyspaceMetadata::getName).collect(Collectors.toSet());
+        result.removeAll(internals);
+        return result.stream().map(KeyspaceImpl::of).collect(Collectors.toSet());
```

Comment:
minor, but ideally we could just use `filter()` rather than collecting in a set and then re-stream it